src/config/CodeGeneratorConfig.php

```diff
@@ -4,9 +4,9 @@
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
 class CodeGeneratorConfig {
-	
+
 	protected $options;
-	
+
 	public function __construct(array $options = []) {
```

Comment:
What do you mean? Which constructor and return type? You mean the generated type hints for `__construct()` and `__destruct()` methods?